packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html

```diff
@@ -1,17 +1,25 @@
 <template>
-  <button aria-hidden={isQuickviewOpen} class={buttonClass} onclick={openQuickview} tooltip={buttonLabel} title={buttonLabel} disabled={hasNoPreview} aria-label={buttonAriaLabelValue}>
-    <template if:true={hasButtonLabel}>
-      {previewButtonLabel}
-    </template>  
-    <template if:true={hasIcon}>
-      <lightning-icon 
-      size="x-small"
-      icon-name={previewButtonIcon}
-      alternative-text={buttonLabel}
-      class={buttonIconClass}>
-      </lightning-icon>
+  <div class="slds-is-relative slds-show_inline result-action_container">
+    <button aria-hidden={isQuickviewOpen} class={buttonClass} onclick={openQuickview} title={buttonTitle} disabled={hasNoPreview} aria-label={buttonAriaLabelValue}>
```

Comment:
Prettier doesn't ask you to put this on multiple lines?

Reply:
I think our prettier config ignores this as it couldn't parse it. maybe it can now idk

Reply:
It's artisanal